src/main/resources/webapp/partials/modals/visitRescheduleModal.html

```diff
@@ -13,13 +13,25 @@ <h4 class="modal-title">{{rescheduleModalHeader}}</h4>
                     </div>
                     <div class="prevac input-group">
                         <span class="left input-group-addon">{{msg('prevac.visitReschedule.plannedDate')}}</span>
-                        <input id="plannedDateInput" type="text" class="form-control datepicker" readonly='true' prevac-date-picker min="form.dto.minDate" max="form.dto.maxDate" placeholder="{{msg('prevac.datePlaceholder')}}" ng-model="form.dto.plannedDate">
-                        <span class="input-group-addon validator" ng-class="{'alert-danger': form.dto.plannedDate === '', 'alert-success': form.dto.plannedDate}"><i class="fa fa-fw fa-check"></i></span>
+                        <input id="plannedDateInput" type="text" class="form-control datepicker" ng-if="showPlannedDate()" readonly='true' prevac-date-picker min="form.dto.minDate" max="form.dto.maxDate" placeholder="{{msg('prevac.datePlaceholder')}}" ng-model="form.dto.plannedDate">
+                        <input id="plannedDateTextInput" type="text" class="form-control" ng-if="!showPlannedDate()" readonly='true' ng-model="form.dto.plannedDate">
```

Comment:
did you try to use ng-disabled?

Reply:
Yes I did but this attribute is not working correctly if attribute readonly is used.